### PR TITLE
main content day 1

### PR DESCRIPTION
**Pull request by jabmat**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +1,77 @@
...
+            </a>
+            <div class="header-top-right"></div>
+        </div>
+
+        <div class="header-title">
+            <h1>No Escape</h1>
+        </div>
+        <!-- anchors for social media "buttons" img's -->
+        <!-- zamienić imgs na <i> z klasą iconmoon oraz dodać klasę od wielkości fontu do wyświetlenia -->
+        <div class="header-social-icons">
```
Comment (Bopiotr) [inline index.html]:
```suggestion
        <div class="header-icons">
```

File: index.html [hunk 2]
```diff
@@ -1,14 +1,77 @@
...
+            <a class="header-social-anchor"
+                href="https://open.spotify.com/artist/50rAhktQ7N9PqCJc7gEI0h?si=VfYSl7QJTryUbbyt3t_ldw" target="_blank">
+                <i class="icon-spotify icon-social"></i>
+            </a>
+            <a class="header-social-anchor" href="https://intodark.bandcamp.com/releases" target="_blank">
+                <i class="icon-bandcamp icon-social"></i>
+            </a>
+            <a class="header-social-anchor" href="https://www.youtube.com/@intodark2552/featured" target="_blank">
```
Comment (Bopiotr) [inline index.html]:
```suggestion
            <a class="header-link" href="https://www.youtube.com/@intodark2552/featured" target="_blank">
```

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+/* 2nd */
+.play-button {
+	animation: fadeIn 10s;
+	-webkit-animation: fadeIn 10s;
+	-moz-animation: fadeIn 10s;
+	-o-animation: fadeIn 10s;
+	-ms-animation: fadeIn 10s;
+}
+
+@keyframes fadeIn {
```
Comment (Bopiotr) [inline styles.css]: remove duplicates

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+	/* align-left icon in button container */
+	.icon-play {
+		float: left;
+	}
+
+	/* effects turning off */
+	.play-button:hover,
+	a.header-social-anchor:hover {
+		transform: none;
```
Comment (Bopiotr) [inline styles.css]: to nie potrzebne

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+	}
+
+	.header-logo-anchor:hover,
+	a.mail-contact:hover {
+		opacity: none;
```
Comment (Bopiotr) [inline styles.css]: to samo

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+	font-weight: 400;
+	font-size: 14px;
+}
+
+/* ---footer--- */
+
+/* Media Query for Mobile Devices */
+@media (max-width: 480px) {
+	.header-top {
+		justify-content: unset;
```
Comment (Bopiotr) [inline styles.css]:
```suggestion
		justify-content: center;
		postinon: relative
```

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+	}
+
+	/* empty right header div -  display default on */
+	.header-top-right {
```
Comment (Bopiotr) [inline styles.css]: remove this element

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,317 @@
...
+@-ms-keyframes fadeIn {
+	0% {
+		opacity: 0;
+	}
+	100% {
+		opacity: 1;
+	}
+}
+/* 3rd */
+.header-title {
```
Comment (Bopiotr) [inline styles.css:39]: move to .header-title above

**Review by Bopiotr — CHANGES_REQUESTED** [review]

**Comment by jabmat** [comment]
I've applied & pushed suggested changes @Bopiotr

**Comment by Bopiotr** [comment]
Please change 'Into Dark' logo to .webp format. And question why do we need two same pictures?
image

**Review by Bopiotr — APPROVED** [review]

**Pull request merged**